onward/app/controllers/RelatedController.scala

```diff
@@ -25,32 +25,20 @@ object RelatedController extends Controller with Related with Containers with Lo
   def render(path: String, isMf2: Boolean = false) = MemcachedAction { implicit request =>
     val edition = Edition(request)
     val excludeTags = request.queryString.getOrElse("exclude-tag", Nil)
-    related(edition, path, excludeTags) map {
+    val relatedType = request.queryString.getOrElse("related-type", Nil)
```

Comment:
you could put .headOption on the end of this and treat it as an Option[String] afterwards (which is basically what it is in reality)